package-lock.json

```diff
@@ -15727,7 +15719,8 @@
     "wrappy": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
-      "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8="
+      "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=",
+      "dev": true
```

Comment:
Any idea if this `"dev": true` is intentional?

Reply:
yep it was as part of my inline change in the webapp
https://github.com/mattermost/mattermost-webapp/pull/3026/files#diff-32607347f8126e6534ebc7ebaec4853dL15715